sw/device/silicon_creator/lib/crypto/ecdsa_p256/meson.build

```diff
@@ -11,3 +12,96 @@ sw_lib_crypto_otbn_sources += {
     'p256.s',
   ),
 }
+
+# Setup for building OTBN assembly. OTBN software is build with a separate
+# toolchain, which is called as an external target from Meson in
+# subdirectories.
+
+prog_otbn_build = meson.source_root() / 'util/otbn_build.py'
+
+otbn_build_command = [
+  prog_env,
+  'OTBN_AS=@0@'.format(prog_otbn_as),
+  'OTBN_LD=@0@'.format(prog_otbn_ld),
+  'RV32_TOOL_OBJCOPY=@0@'.format(prog_objcopy.path()),
+  'RV32_TOOL_AS=@0@'.format(prog_as.path()),
+  'RV32_TOOL_LD=@0@'.format(prog_ld.path()),
+  prog_python,
+  prog_otbn_build,
+  '--out-dir',
+  '@OUTDIR@',
+  '@INPUT@',
+]
```

Comment:
`prog_otbn_build`,  `otbn_build_command` and `otb_build_depend_files` can be moved to the `lib/crypto/meson.build` file before including subdirs.